spec/factories/users.rb

```diff
@@ -5,6 +5,7 @@
     password { 'passw0rd' }
     sequence(:first_name) { |n| "First_#{n}" }
     last_name { 'Last' }
+    gender { 0 }
```

Comment:
This is the line that it claimed was outdated; is there a reason why we're using the integer value and not the string here?